### DISCUSSION TITLE
[RTSP source] received RTP packet with unknown payload type: 8

### DISCUSSION THREAD

**Discussion by dota2015** [bluenviron/mediamtx#2788] [Questions]
### Question

When using RTSP to HLS conversion with H.265, an error is encountered: "[RTSP source] received RTP packet with unknown payload type: 8." Is there any solution to this issue?

**Comment by u-nine** [comment]
![image](https://github.com/bluenviron/mediamtx/assets/12064578/b515d365-da24-4577-9244-4c9f8deb7fd9)

**Comment by aler9** [comment]
Hello, please attach a server log with option `logLevel` set to `debug`, otherwise it's not possible to further dig into the problem.

**Comment by pikachu937** [comment]
Greetings

There is a similar problem
received RTP packet with unknown payload type: 50 is constantly falling in the log

[mtx.tar.gz](https://github.com/bluenviron/mediamtx/files/14371807/mtx.tar.gz)

**Comment by aler9** [comment]
If you want to get this issue fixed, open a bug report and provide a server log and network dump in order to allow replicating the problem.
